augur/__init__.py

```diff
@@ -75,6 +75,9 @@ def run(argv):
     args = make_parser().parse_args(argv)
     try:
         return args.__command__.run(args)
+    except SystemExit as e:
```

Comment:
For my own understanding, this lets code which (e.g.) printed it's own error message & called `sys.exit(2)` to work as intended and avoid the `except:` block below?

Reply:
Yeah, well any `sys.exit` call will be let through here. I see a mix of `sys.exit(2)` and `sys.exit(1)` used in Augur. Not sure what the standard is, but they all seem to be for expected errors.

I was prompted to add this block by [these failing tests](https://github.com/nextstrain/augur/runs/6315719620?check_suite_focus=true), which you can look at for understanding.

Reply:
I've removed the `except SystemExit` per @tsibley's suggestion: 65b7b4a1

Reply:
Created a separate issue #917